code/game/machinery/status_display.dm

```diff
@@ -220,7 +220,7 @@
 	return "[add_zero(num2text((timeleft / 60) % 60),2)]:[add_zero(num2text(timeleft % 60), 2)]"
 
 /obj/machinery/status_display/proc/get_supply_shuttle_timer()
-	var/datum/shuttle/autodock/ferry/supply/shuttle = supply_controller.shuttle
+	var/datum/shuttle/autodock/ferry/supply/shuttle = SSsupply.shuttle

```

Comment:
Is this going to make the display work like it used to? As in show how long is left before supply shuttle gets to the station, as happened before the original refactor of the supply system into using NanoUI for its console.

Reply:
No, the only behaviors this PR will change is the frequency of supply points (Still 3/minute, but distributed as 1 every 20s instead of 1.5 every 30s)
The issue you are referring to is due to a bug in the supply computer.  I could fix that at the same time but wasn't sure if it would be confusing to include in this PR?

Reply:
Ah, alright. Thanks for your answer. :> And I'd think that'd have to go in a different PR, yeah.

Reply:
Can you make a bug report for that? Because just looking at the code below here I can't see a reason why it shouldn't work.

Reply:
@Atermonera Issue PolarisSS13/Polaris#6910 added.